src/table/body-cell/styles.scss

```diff
@@ -147,28 +148,21 @@ $cell-negative-space-vertical: 2px;
   }
   &:first-child {
     border-inline-start: $border-placeholder;
+    @include cell-padding-inline-start($cell-edge-horizontal-padding);
   }
   &:last-child {
     border-inline-end: $border-placeholder;
     @include cell-padding-inline-end($cell-edge-horizontal-padding);
   }
+  // Using very small padding for 1st column cells in VR.
   &.is-visual-refresh:first-child {
-    &:not(.has-striped-rows) {
```

Comment:
Removed "&:not(.has-striped-rows)" because there is an explicit override for ".has-striped-rows" down below.